code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -20,80 +20,46 @@ adjustFireLoss(-(maxHealth / 70 + 0.5 + (maxHealth / 70) * recovery_aura/2)*(m))
 
 	..()
 
-	switch(stat)
-		if(DEAD) //Dead, nothing else to do.
-			return
-		if(UNCONSCIOUS)
-			if(is_zoomed)
+	if(stat == DEAD) //Dead, nothing else to do.
+		return
+	if(stat == UNCONSCIOUS)
+		if(is_zoomed)
+			zoom_out()
+	else
+		if(is_zoomed)
+			if(loc != zoom_turf || lying)
 				zoom_out()
-			handle_aura_receiver()
-			handle_living_status_updates()
-			handle_living_health_updates()
-			handle_living_plasma_updates()
-			handle_critical_status_updates()
-			update_canmove()
-			update_icons()
-		if(CONSCIOUS)
-			if(is_zoomed)
-				if(loc != zoom_turf || lying)
-					zoom_out()
-			update_progression()
-			update_evolving()	
-			//Status updates, death etc.
-			handle_aura_emiter()
-			handle_aura_receiver()
-			handle_living_status_updates()
-			handle_living_health_updates()
-			handle_living_plasma_updates()
-			handle_conscious_status_updates()
-			update_canmove()
-			update_icons()
-			if(client)
-				handle_regular_hud_updates()
-
-/mob/living/carbon/Xenomorph/proc/handle_critical_status_updates()
-	if(health > 0)
-		if(!knocked_out && !sleeping) //wake if not napping
-			blinded = FALSE
-			stat = CONSCIOUS
-			see_in_dark = 8
-			ear_deaf = 0 //All this stuff is prob unnecessary
-			ear_damage = 0
-			eye_blind = 0
-			eye_blurry = 0
+		update_progression()
+		update_evolving()
+		handle_aura_emiter()
```

Comment:
Unconscious xenos shouldn't emit aura, and no need for them to age or update evolving.

Reply:
You might be reading the code wrongly, this is properly indented under else (alias: CONSCIOUS).